__tests__/app.test.js

```diff
@@ -225,28 +225,103 @@ describe("express app", () => {
       });
     });
   });
-  describe('GET /api/articles', () => {
-    it('status 200: with array of articles', async () => {
-      const {body: {articles}} = await request(app).get("/api/articles")
-      articles.forEach((article)=> expect(article).toEqual(
-        expect.objectContaining({
-          article_id: expect.any(Number),
-          title: expect.any(String),
-          topic: expect.any(String),
-          author: expect.any(String),
-          body: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-          comment_count: expect.any(Number)
-      })
-    ))
+  describe.only("GET /api/articles", () => {
+    it("status 200: with array of articles", async () => {
+      const {
+        body: { articles },
+      } = await request(app).get("/api/articles");
+      articles.forEach((article) =>
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          })
+        )
+      );
     });
-    it("status 200: sorted in descending order", async () => {
+    it("status 200: sorted by created_at in descending order by default", async () => {
       const {
         body: { articles },
       } = await request(app).get("/api/articles");
       expect(articles).toBeSortedBy("created_at", { descending: true });
     });
+    it("status 200: accepts alternate sort_by queries ", async () => {
+      const columns = [
+        "title",
+        "topic",
+        "author",
+        "body",
+        "created_at",
+        "votes",
+      ];
+      for (let i = 0; i < columns.length; i++) {
+        const {
+          body: { articles },
+        } = await request(app)
+          .get("/api/articles")
+          .query({ sort_by: columns[i] });
+        expect(articles).toBeSortedBy(columns[i], {
+          descending: true,
+          coerce: true,
```

Comment:
adding the coerce flag to this will cause unexpected behaviour. According to the docs, if this is set to true, any values which cannot coerce to a number will be instead be coerced to NaN and is considered sorted.